commands/uploader.go

```diff
@@ -289,6 +289,11 @@ func (c *uploadContext) ReportErrors() {
 		}
 
 		if !c.allowMissing {
+			pushMissingHint := []string{
+				"hint: Your push was rejected due to missing or corrupt local objects.",
+				"hint: Overwrite the reject with `git config lfs.allowincompletepush true`.",
```

Comment:
Wording nit: Maybe something along the lines of

> hint: You can disable this check using...

Reply:
Fixed 👍 
Following: https://github.com/git-lfs/git-lfs/blob/f5f57312610a86eb1fdb9f1cce4c071d589ccce2/commands/lockverifier.go#L66